doc/src/arch/reference.rst

```diff
@@ -595,6 +595,11 @@ The tags within the ``<device>`` tag are:
     Content inside this tag is only used when VPR is in global routing mode.
     The contents of this tag are described in :ref:`global_routing_info`.
 
+.. arch:tag:: <default_fc in_type="{frac|abs}" in_val="{int|float}" out_type="{frac|abs}" out_val="{int|float}"/>
+
+    This defines the default Fc specification, if it is not specified within a ``<fc>`` tag inside a top-level complex block.
+    The attributes have the same meaning as within a ``<fc>`` tag.
```

Comment:
If we're going to reference the `<fc>` tag for the attributes, it would be good to include a click-able link (`:ref:`) to the `<fc>` tag.